### DISCUSSION TITLE
File missing `path` - how to turn into string?

### DISCUSSION THREAD

**Discussion by ScottAgirs** [outline/rich-markdown-editor#472] [Q&A]
I am trying to initiate a `new FileReader()` object from the file `Blob` that's provided by the `uploadImage` callback, but it keeps failing, it seems that the Blob does not contain the path. Could anyone help me navigate this please?

How can I create a string from the Blob that's provided with the `uploadImage` callback?

**Comment by ScottAgirs** [comment]
@tommoor pardon the direct ping, but any chance u have some pointers here?

**Reply by tommoor** [reply]
The `file` object that is passed to `uploadImage` is directly from a file input, it should have exactly the same properties as though you created something like `<input type="file" onchange={uploadImage} />`.

The specifics of FileReader I'm not sure about, would probably consult MDN documentation
